scripts/import_tokens.sh

```diff
@@ -2,18 +2,18 @@
 
 set -e
 
-SHORT=b,:
-LONG=branch,:
-BRANCH='main'
+SHORT=t,:
+LONG=tag,:
+TAG='v1.4.0'
```

Comment:
I think that to properly handle https://element-io.atlassian.net/browse/PSB-700 we should just have an hard-coded value of the tag somewhere and not allow passing it as command line parameter, to avoid any mistake. WDYT @giomfo